server/src/test/scala/services/ArrivalSpec.scala

```diff
@@ -164,4 +165,77 @@ class ArrivalSpec extends Specification {
       result === flightsInRange
     }
   }
+
+  "Given an arrival" >> {
+    "When flight Actual Chox time exists" >> {
+      "display status should be On Chocks" >> {
+        val arrival: Arrival = ArrivalGenerator.arrival(actPax = Option(10), actDt = "2020-10-22T13:00Z", actChoxDt = "2020-10-22T13:00Z")
+        val result = arrival.displayStatus
+        result === ArrivalStatus("On Chocks")
+      }
+    }
+    "When flight Actual time exists and Actual Chox time does not exists" >> {
+      "display status should be Landed" >> {
+        val arrival: Arrival = ArrivalGenerator.arrival(actPax = Option(10), actDt = "2020-10-22T13:00Z")
+        val result = arrival.displayStatus
+        result === ArrivalStatus("Landed")
+      }
+    }
+    "When flight Estimated Arrival is no more than 15 minutes later than scheduled time" >> {
+      "display status should be Delayed" >> {
+        val arrival: Arrival = ArrivalGenerator.arrival(actPax = Option(10), schDt = "2020-10-22T13:00Z", estDt = "2020-10-22T13:20Z")
+        val result = arrival.displayStatus
+        result === ArrivalStatus("Delayed")
+      }
+    }
+    "When flight Estimated Arrival is more than 15 minutes later than scheduled time" >> {
```

Comment:
This description is inverted. It should say '.. not more than ..' or ' .. less than ..'